source/MRVoxels/MRVDBConversions.h

```diff
@@ -61,9 +61,14 @@ MRVOXELS_API FloatGrid simpleVolumeToDenseGrid( const SimpleVolume& simpleVolume
 // set the simpleVolume.min as the background value
 MRVOXELS_API VdbVolume simpleVolumeToVdbVolume( const SimpleVolumeMinMax& simpleVolume, ProgressCallback cb = {} );
 
-/// Copy given \p simpleVolume into the \p grid, starting at \p minCoord
+/// @brief Copy given \p simpleVolume into the \p grid, starting at \p minCoord
+/// Instantiated for AccessorOrGrid in { openvdb::FloatGrid::Accessor, FloatGrid, openvdb::FloatGrid }.
+/// The template is used to not include openvdb's mess into this header (forward declaring classes in openvdb is also non-trivial).
+/// When used with a Grid, multithreaded implementation of copying is used (so the function is not thread safe).
+/// When used with an Accessor, this function could be called from different threads on the same volume (provided that accessors are different, of course).
+template <typename AccessorOrGrid>
```

Comment:
may be create 2-3 non-template functions instead?

Reply:
Then we'll have to either include openvdb's headers (increasing compile time) or forward declare a lot of openvdb's structures (to decalre an `Accessor`), which is not trivial